### PR TITLE
Opt in to per-monitor DPI awareness

### PR DESCRIPTION
**Pull request by AgmasGold**
This PR opts-in to per-monitor DPI awareness.
https://github.com/Microsoft/WPF-Samples/tree/master/PerMonitorDPI

Its not perfect on my machine, but its better than the old blurry text. When I move the window from my 4k monitor to a 1080p monitor, the window title bar extends to double height, and context menu text is much larger:
Updated:
image

Original:
image

The blurry text does not show up on the images, but the overall result despite the double height title bar is much more bearable.

Additionally, this also tweaks the editor config to remove the "Object initialisation can be simplified" message (as I'm fine with either method, and it saves no lines of code nor makes anything clearer), and tweaks the App.xaml.cs file (just using pattern matching where suggested and simplifying the using statement).

### CODE REVIEW

File: AnnoDesigner/app.manifest [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+  <!-- Indicates that the application is DPI-aware and will not be automatically scaled by Windows at higher
+       DPIs. Windows Presentation Foundation (WPF) applications are automatically DPI-aware and do not need 
+       to opt in. Windows Forms applications targeting .NET Framework 4.6 that opt into this setting, should 
+       also set the 'EnableWindowsFormsHighDpiAutoResizing' setting to 'true' in their app.config. -->
+  
+  <application xmlns="urn:schemas-microsoft-com:asm.v3">
+    <windowsSettings>
+      <dpiAwareness xmlns="http://schemas.microsoft.com/SMI/2016/WindowsSettings">PerMonitor</dpiAwareness>
+      <dpiAware xmlns="http://schemas.microsoft.com/SMI/2005/WindowsSettings">true</dpiAware>
```
Comment (FroggieFrog) [inline AnnoDesigner/app.manifest:56]: should be `true/pm` (https://docs.microsoft.com/en-us/windows/win32/sbscs/application-manifests#dpiaware)

File: AnnoDesigner/app.manifest [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  </compatibility>
+
+  <!-- Indicates that the application is DPI-aware and will not be automatically scaled by Windows at higher
+       DPIs. Windows Presentation Foundation (WPF) applications are automatically DPI-aware and do not need 
+       to opt in. Windows Forms applications targeting .NET Framework 4.6 that opt into this setting, should 
+       also set the 'EnableWindowsFormsHighDpiAutoResizing' setting to 'true' in their app.config. -->
+  
+  <application xmlns="urn:schemas-microsoft-com:asm.v3">
+    <windowsSettings>
+      <dpiAwareness xmlns="http://schemas.microsoft.com/SMI/2016/WindowsSettings">PerMonitor</dpiAwareness>
```
Comment (FroggieFrog) [inline AnnoDesigner/app.manifest:55]: should be `PerMonitorV2, PerMonitor` (https://docs.microsoft.com/en-us/windows/win32/sbscs/application-manifests#dpiawareness)

File: AnnoDesigner/app.manifest [hunk 3]
```diff
@@ -0,0 +1,77 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<assembly manifestVersion="1.0" xmlns="urn:schemas-microsoft-com:asm.v1">
+  <assemblyIdentity version="1.0.0.0" name="MyApplication.app"/>
```
Comment (FroggieFrog) [inline AnnoDesigner/app.manifest:3]: maybe should be adjusted

File: AnnoDesigner/app.manifest [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+            Remove this element if your application requires this virtualization for backwards
+            compatibility.
+        -->
+        <requestedExecutionLevel level="asInvoker" uiAccess="false" />
+      </requestedPrivileges>
+    </security>
+  </trustInfo>
+
+  <compatibility xmlns="urn:schemas-microsoft-com:compatibility.v1">
+    <application>
```
Comment (FroggieFrog) [inline AnnoDesigner/app.manifest:25]: Maybe uncomment all the `supportedOS` elements?

File: AnnoDesigner/App.xaml.cs [hunk 5]
```diff
@@ -117,107 +132,98 @@ private async void Application_Startup(object sender, StartupEventArgs e)
...
+                {
+                    File.Delete(fileName);
+                }
 
-                IRecentFilesHelper recentFilesHelper = new RecentFilesHelper(recentFilesSerializer, _fileSystem);
-                var mainVM = new MainViewModel(_commons, _appSettings, recentFilesHelper, _messageBoxService, _updateHelper, _localizationHelper, _fileSystem);
+                _appSettings.Reload();
+            }
 
-                //TODO MainWindow.ctor calls AnnoCanvas.ctor loads presets -> change logic when to load data
```
Comment (FroggieFrog) [inline AnnoDesigner/App.xaml.cs:198]: I think this comment still reflects the current state.

Comment (AgmasGold) [reply AnnoDesigner/App.xaml.cs:198]: I'll get that added back in then

File: AnnoDesigner/app.manifest (hunk 3 above)
Comment (AgmasGold) [reply AnnoDesigner/app.manifest:3]: That was just the default way the file was created when I did "Add new item" from visual studio.

File: AnnoDesigner/app.manifest (hunk 4 above)
Comment (AgmasGold) [reply AnnoDesigner/app.manifest:25]: That was just the default way the file was created when I did "Add new item" from visual studio. Personally I would just leave it as is, as I'm not sure what the full ramifications of uncommenting them may be.

File: AnnoDesigner/app.manifest (hunk 2 above)
Comment (AgmasGold) [reply AnnoDesigner/app.manifest:55]: Good catch

File: AnnoDesigner/app.manifest (hunk 1 above)
Comment (AgmasGold) [reply AnnoDesigner/app.manifest:56]: good catch.

File: AnnoDesigner/app.manifest (hunk 3 above)
Comment (AgmasGold) [reply AnnoDesigner/app.manifest:3]: Ah - nevermind - just seen what you saw...

**Review by FroggieFrog — APPROVED** [review]

File: AnnoDesigner/app.manifest (hunk 3 above)
Comment (AgmasGold) [reply AnnoDesigner/app.manifest]: Updated

**Pull request merged**